sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -130,7 +201,19 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
    *
    * @param options - Optional parameters. See {@link DefaultAzureCredentialOptions}.
    */
-  constructor(options?: DefaultAzureCredentialOptions) {
+  constructor(options?: DefaultAzureCredentialOptions);
+
+  /**
+   * Creates an instance of the DefaultAzureCredential class.
+   * @internal
+   * @hidden
+   */
```

Comment:
Strictly speaking you don't need documentation, `@hidden`, or `@internal` on this because it is the implementation of the constructor and only the overload signatures actually appear in docs.